XIVComboExpanded/CustomComboPreset.cs

```diff
@@ -1330,10 +1330,18 @@ public enum CustomComboPreset
 
     [IconsCombo([NIN.Kassatsu, UTL.ArrowLeft, NIN.TrickAttack, UTL.Blank, UTL.Blank, NIN.Buffs.ShadowWalker, NIN.Buffs.Hidden, UTL.Checkmark])]
     [SectionCombo("Ninjutsu features")]
+    [ConflictingCombos(NinjaDreamyTrickAttack)]
     [ExpandedCustomCombo]
     [CustomComboInfo("Kassatsu to Trick", "Replace Kassatsu with Trick Attack while Suiton or Hidden is up.\nCooldown tracking plugin recommended.", NIN.JobID)]
     NinjaKassatsuTrickFeature = 3004,
 
+    [IconsCombo([NIN.Kassatsu, UTL.ArrowLeft, NIN.TrickAttack, UTL.ArrowLeft,  NIN.DreamWithinADream])]
+    [SectionCombo("Ninjutsu features")]
+    [ConflictingCombos(NinjaKassatsuTrickFeature)]
+    [SecretCustomCombo]
+    [CustomComboInfo("Optimised Kassatsu Trick", "Replace Kassatsu With Trick only while Suiton is up and Kassatsu Is off cooldown. Also replaces with Dream within a dream if both on cooldown.", NIN.JobID)]
+    NinjaDreamyTrickAttack = 3093,
```

Comment:
```suggestion
    [IconsCombo([NIN.Kassatsu, UTL.ArrowLeft, NIN.TrickAttack, UTL.ArrowLeft,  NIN.DreamWithinADream])]
    [SectionCombo("Ninjutsu features")]
    [ParentCombo(NinjaKassatsuTrickFeature)]
    [SecretCustomCombo]
    [CustomComboInfo("Optimised Kassatsu Trick", "Replace Kassatsu with Trick only while Shadow Walker is up and Kassatsu is on cooldown. Also replaces Kassatsu with Dream Within a Dream if both it and Trick Attack are on cooldown.", NIN.JobID)]
    NinjaDreamyTrickAttack = 3093,
```

Recommended rewording.  Also, shifting to parented on the existing Kassatsu/Trick Attack combo, rather than conflicting, as I recommended in a prior comment.